packages/core/src/components/button/button.stories.ts

```diff
@@ -0,0 +1,42 @@
+import { htmlMatrix, Meta, Story } from '../../../../../docs';
+import { Icon } from '../icon/icon.story';
+import { Button, HtmlButtonProps } from './button.story';
+
+export default {
+  title: 'Core/Button',
+  component: Button,
+  args: {
+    children: 'Button',
+    disabled: false,
+    kind: 'action',
+    variant: 'primary',
+  },
+} as Meta<HtmlButtonProps>;
+
+export const Playground: Story<HtmlButtonProps> = (props) => Button(props);
+
+export const AsAnchor: Story<HtmlButtonProps> = (props) =>
+  Button({ ...props, href: 'javascript:0' });
```

Comment:
This does not work, probably better to set as:

```suggestion
  Button({ ...props, href: '#' });
```

But then (in general for a button) disable `onClick`?

Reply:
What doesn't work? It renders as `<a>` correctly and you can click it (has `:hover`, `:active` and other pseudos).

The annoying part about disabling `onClick` is that it shows on the code snippet, which I don't particularly like.

Reply:
Yes, but when I click it I end up on an empty page with `0` rendered 😓 (on Firefox).

Reply:
Oh Firefox, you silly goose. `javascript: void 0;` should do the trick then.

```suggestion
  Button({ ...props, href: 'javascript:void 0' });
```

Reply:
Did it work for you? Does not for me...

Reply:
For some reason the CI was not working yesterday, GitHub actions were down or something.
Try now.

Reply:
Yep, now it's all fine.

Probably would also be good to adjust React story too...

Reply:
Done.